### PR TITLE
Allow IDMS statements in conditional block #952

### LINKED ISSUE #952: Syntax errors when IDMS ready statements are within an IF block
## Description ##
IDMS ready statements have syntax errors when inside an if block. This should be valid syntax as IDMS uses `;` instead of `.` to end generated ready statement code within an if block.

## Screenshots ##
image

## Environment ##
 - OS: Windows 10, Manjaro Linux
 - IDE: VS Code

### PR DESCRIPTION
**Pull request by temanbrcom**
Fix #952

### CODE REVIEW

File: server/src/test/java/org/eclipse/lsp/cobol/usecases/TestIdmsReadyAfterIfNotProducesError.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2020 Broadcom.
```
Comment (asatklichov) [inline server/src/test/java/org/eclipse/lsp/cobol/usecases/TestIdmsReadyAfterIfNotProducesError.java:2]: 2021

Comment (temanbrcom) [reply server/src/test/java/org/eclipse/lsp/cobol/usecases/TestIdmsReadyAfterIfNotProducesError.java:2]: 😮

**Review by asatklichov — APPROVED** [review]
🍔

**Review by grianbrcom — APPROVED** [review]
:safety_pin:

**Comment by grianbrcom** [comment]
Please, rebase it.

**Pull request merged**